src/Location/WorkingSchedule.php

```diff
@@ -320,7 +320,7 @@ public function getTimeslot(int $interval = 15, DateTime $dateTime = null, int $
         $dateTime = Carbon::instance($this->parseDate($dateTime));
         $checkDateTime = $dateTime->copy();
         $interval = new DateInterval('PT'.($interval ?: 15).'M');
-        $leadTime = new DateInterval('PT'.($leadTime ?: 25).'M');
+        $leadTimeInterval = new DateInterval('PT'.($leadTime ?: 25).'M');
```

Comment:
Not necessary as you can also access the value through `$leadTime->i`

Reply:
Changed :)